core/src/main/java/org/elasticsearch/indices/IndicesQueryCache.java

```diff
@@ -49,7 +49,7 @@
 
 public class IndicesQueryCache extends AbstractComponent implements QueryCache, Closeable {
 
-    public static final Setting<ByteSizeValue> INDICES_CACHE_QUERY_SIZE_SETTING = Setting.byteSizeSetting(
+    public static final Setting<ByteSizeValue> INDICES_CACHE_QUERY_SIZE_SETTING = Setting.memorySizeSetting(
```

Comment:
Same thing here, move the invocation to the next line with continuation indentation so that the invocation and its parameters are together?